DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,14 @@ public function getConfigTreeBuilder()
         $rootNode
         ->children()
             ->scalarNode('output_file_prefix')->defaultValue(gethostname())->end()
+            ->arrayNode('error_notification')
+                ->children()
+                    ->scalarNode('from')->defaultValue('dizda@backupbundle.com')->end()
+                    ->scalarNode('to')->defaultValue('')->end()
+                    ->scalarNode('subject')->defaultValue('Backup Error')->end()
+                    ->scalarNode('body')->defaultValue('...')->end()
```

Comment:
I think this node is useless.
You might specify an existing content with the detail of the exception, what do you think?